### ISSUE TITLE
Weird color behavior in Vim as of commit 13a0b6bb

### ISSUE THREAD

**Issue by wincent** [tmux/tmux#776]
I just tried the latest tmux `master` and found that the colors in my Vim instance go crazy as soon as Vim loses focus. It's possible there's a quirk in my set-up and I may be doing something wrong but I'm reporting it anyway in case something unintended happened on the tmux side.

This screenshot shows the behavior; Vim is on the left and is bright green, but it's supposed to be dark, almost black.

<img width="720" alt="__tmux__tmux_" src="https://cloud.githubusercontent.com/assets/7074/22912287/5fcafbc2-f219-11e6-8c9c-136aa14e2ff8.png">

I bisected the problem to 13a0b6bb. Prior to that commit, if behaves as expected.

This is on Darwin i386, `$TERM` is `xterm-256color` outside of tmux, and `tmux-256color` inside tmux.

I repro'd with `./tmux -vvvv -Ltest -f/dev/null`. Then I ran `export TERM=tmux-256color`, ran `vim`, and created a split; as soon as Vim loses focus the colors get borked.

This only happens with `tmux-256color`, not with `screen` or `tmux` or `screen-256color`. The terminfo for `tmux-256color` was made with `tic -o path/to/terminfo path/to/input` and this input file:

```
tmux-256color|tmux with 256 colors,
  sitm=\E[3m, ritm=\E[23m,
  smso=\E[7m, rmso=\E[27m,
  use=screen-256color,
```

Will attach the server log file:

[tmux-server-63158.log.zip](https://github.com/tmux/tmux/files/772978/tmux-server-63158.log.zip)

Like I said, I may be doing something wrong. Thanks for any light you can shed on this. In the meantime, I'm going to run on the parent commit, which doesn't exhibit the problem.

**Comment by nicm** [comment]
I am pretty sure this is fixed but the commit has not made it to GitHub yet, if you wait a day or so it should turn up.

**Comment by wincent** [comment]
Good to know. I'll loop back in a few days and close this once I've confirmed that it's fixed.

**Comment by wincent** [comment]
Actually, looks like it just got merged in (guessing 4c2a78029d0cb307899be42d94d3c255b97fb15f was the fix). Thanks!
